plugin/rollback.go

```diff
@@ -297,11 +302,13 @@ func isGoogleApiErrorWithCodes(err error, validErrCodes ...int) bool {
 	if err == nil {
 		return false
 	}
-	gErr, ok := err.(*googleapi.Error)
+	ok := errwrap.ContainsType(err, new(googleapi.Error))
```

Comment:
Curious on this, was there an issue with type assertion on the error?

Reply:
So, that was necessary because the method to grab the iam_policy was wrapped with errwrap. By doing so, it always returned false. So I needed to check all of the errors to make sure they weren't googleapi errors that were wrapped, and then grab them and cast them as googleapi errors.